docs/en/observability/whats-new.asciidoc

```diff
@@ -3,15 +3,118 @@
 
 // What's new content goes in here. Don't uncomment or remove the tags surrounding this content :)
 
+[discrete]
+== New Synthetics App enhancements for public beta
+
+As we progress towards delivering an all new Synthetic Monitoring capability
+within Elastic Observability, we’re excited to announce two big new enhancements
+for our 8.4 stack release:
+
+- Private testing locations
+- Open access to the managed testing service for all Elastic Cloud users
+
+With private testing locations you can now configure a testing location on
+self-managed infrastructure. This means that you can test applications that
+aren’t publically available on the internet or from locations where we may not
+yet have a point of presence.
+
+A big part of our new Synthetics app is the ability to run synthetic monitoring
+tests on our new global network of managed testing locations. With the 8.4
+release, we’ll no longer be requiring users to request access. If you want to
+use our managed testing infrastructure, simply open the {uptime-app} and click
+*Monitor Management*. There is a fair usage cap in place but all test executions
+are free. We’d love to get you involved and get your feedback.
+
+
+[discrete]
+== Observability alerting and rules views now generally available (GA)
+
+Alerting and rule management for Observability-specific alerts and rules have
+been offered as a technical preview since 7.16. We have made significant
+improvements since then and we now have robust, feature-rich
+capabilities that are production ready.
+
+[role="screenshot"]
+image::images/observability-alerts-overview.png[Observability Alerts view]
+
+Working alongside Cases functionality for incident management, Observability
+Alerts and Rules views provide you with an easy way to access and manage
+Observability-specific rules, alerts, and cases within the Observability UI.
+With comprehensive filtering and search support, you can quickly view or edit
+alerts or rules of interest.
+
+[role="screenshot"]
+image::images/observability-rules.png[Observability Rules view]
+
+The Rule Detail view offers rule creators and
+administrators a complete history of rule executions and seamless access to
+generated alerts for effective management. In 8.4 the Rule Details view has been
+further enhanced to include an at-a-glance alert summary that shows the status
+and timeline for triggered alerts, along with a number of other UI improvements,
+including bulk action support.
+
+[role="screenshot"]
+image::images/rules-cpu-exhaustion.png[Observability Rule detail]
+
+For more information on Observability alerting, see {observability-guide}/create-alerts.html[Create alerts].
+
+[discrete]
+== Improved RCA flows for erroneous services
+
+With two new _Top 5_ views introduced for version 8.4, you can quickly identify
+the impact of application errors and reduce time spent identifying and tracking
+errors.
+
+The *Top 5 errors* view is available from the APM Transaction details page:
```

Comment:
We're a bit inconsistent with highlighting in this topic (sometimes the view name is bold, sometimes it is not).

Reply:
Yeah, agreed - I'm also a little confused by what is specifically a view or not when the screenshot doesn't seem to match the supplied text e.g. Rule Detail view, it's a bit of a fudge in places